spec/lib/diaspora/federated/generator_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+describe "comment on reshare" do
+  before do
+    @status_message = FactoryGirl.build(:status_message, author: alice.person, public: true)
+    @reshare = FactoryGirl.build(:reshare, root: @status_message, author: bob.person)
+    @comment = FactoryGirl.build(:comment, post: @reshare, text: "reshare comment")
+  end
+
+  it "delivers comment also to root author" do
+    # opts = {:additional_subscribers => [alice.person]}
+    allow_any_instance_of(Federated::Generator).to receive(:create!)
```

Comment:
OK, I misunderstood `allow` then. Anyway, nothing happens on `@comment.save!` :angry: I'm having difficulty following what should happen but apparently `save!` is not the way. I'll try to find another place to trigger this.